community/en/census/tf20_census.ipynb

```diff
@@ -0,0 +1,1422 @@
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Normalize data for Model Convergence"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "OXAMwr3dCsqd",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "def standardization(dataset):\n",
+        "  \"\"\" Standardization of numeric fields, where all values will have mean of zero \n",
+        "  and standard deviation of one. (z-score)\n",
+        "\n",
+        "  Args:\n",
+        "    dataset: A `Pandas.Dataframe` \n",
+        "  \"\"\"\n",
+        "  dtypes = list(zip(dataset.dtypes.index, map(str, dataset.dtypes)))\n",
+        "  # Normalize numeric columns.\n",
+        "  for column, dtype in dtypes:\n",
+        "      if dtype == 'float32':\n",
+        "          dataset[column] -= dataset[column].mean()\n",
+        "          dataset[column] /= dataset[column].std()\n",
+        "  return dataset"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "078--On5jnpW",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Join all dataframes to standarize data, then split.\n",
+        "all_data = pd.concat([_train_x, _test_x], keys=[0, 1])\n",
+        "standardized_data = standardization(all_data)\n",
+        "train_x, test_x = standardized_data.xs(0), standardized_data.xs(1)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "6NZqE3Wlg1yg",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Verify dataset features\n",
+        "# Note how only the numeric fields (not categorical) have been standardized\n",
+        "train_x.head()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "1iqAXKj6YaBk",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "#### DataFrame Length\n",
+        "\n",
+        "List length of training and testing data. ((32561, 32561, 16281, 16281))"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "BrO4NJfLGfHE",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "len(train_x), len(train_y), len(test_x), len(test_y)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "5VdTmB8zoi8Y",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Dataset input function\n",
+        "\n",
+        "Keras supports tf.data.Dataset, we will be using this new functionality to process the dataset.\n",
+        "\n",
+        "https://www.tensorflow.org/api_docs/python/tf/data/Dataset"
```

Comment:
I think it's better to link to the Dataset guide: https://www.tensorflow.org/guide/datasets